src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -178,6 +178,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.MaxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit", ConfigurationEntryScope.ServerWideOnly)]
         public TimeSetting MaxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit { get; protected set; }
 
+        [Description("Indicates if missing fields should be indexed same as 'null' values or not. Default: false")]
```

Comment:
this is a bad merge?

Reply:
I will handle that and the end, we can ignore this for now